### PR TITLE
Adds Hover Effect to Connections Game

### PR DESCRIPTION
**Pull request by abbiehuynh**
## New Feature: Adds color change hover effect to words that are not disabled.
Word button should change to cyan color when mouse hovers over available word choice as seen with "copy" below. 
Hover Effect

## Impact: 
Allows users to see what word choices are still available. The available word choices will change colors when the mouse hover over it.

## Contribution to
[React Game - Connections](https://github.com/tati-guzman/techtonica-assignments/tree/main/react-game/reactgame)

## Changes made in the following files:
- App.css
- Main.jsx
- Word.jsx

tldr;
Merging Hover Effect Feature into Tati's Main Branch

### CODE REVIEW

**Comment by abbiehuynh** [comment]
Hi Tati, just wanted to let you know that I had to resolve some conflicts with a previous merge. Please let me know if there's anything that needs to be fixed! Otherwise, LGTM!

**Comment by tati-guzman** [comment]
This looks marvelous, thank you for tackling this! Merging your changes now :)
